beacon/goclient/signing.go

```diff
@@ -57,30 +57,16 @@ func (gc *GoClient) computeVoluntaryExitDomain(ctx context.Context) (phase0.Doma
 		CurrentVersion: forkVersion,
 	}
 
-	start = time.Now()
-	genesisResponse, err := gc.multiClient.Genesis(ctx, &api.GenesisOpts{})
-	recordRequestDuration(gc.ctx, "Genesis", gc.multiClient.Address(), http.MethodGet, time.Since(start), err)
+	genesis, err := gc.Genesis(ctx)
 	if err != nil {
 		gc.log.Error(clResponseErrMsg,
 			zap.String("api", "Genesis"),
 			zap.Error(err),
 		)
```

Comment:
We should move the log inside GoClient.Genesis()

Reply:
fixed

Reply:
@olegshmuelov added, please check